jdaviz/configs/default/plugins/subset_plugin/subset_plugin.py

```diff
@@ -54,6 +56,19 @@ def __init__(self, *args, **kwargs):
                                           'subset_items',
                                           'subset_selected',
                                           default_text="Create New")
+        self.all_subsets_with_subregions = {}
```

Comment:
I guess if it works for you, this can be done for now but I feel like it needs to be refactored at some point because we agree that the important algorithm for app should not rest solely within the plugin code.